### PR TITLE
add beer model

### PR DESCRIPTION
**Pull request by mattphoto**
added

### CODE REVIEW

File: models/beers.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  map: { name: 'beerName' },
+  autokey: { path: 'slug', from: 'beerName', unique: true }
+});
+
+Beers.add(  
+  {heading: 'Beer'},
+    {
+      beerTitle: { type: String },
+      beerName: { type: String, required: true, initial: true},
+      brewerName: { type: String, required: true, initial: true},
```
Comment (caitlin-mohnike) [inline models/beers.js]: You should make this a type of name. I assume that will make it easier for us to match ours together. 

Can we have a list of current Brewers and have that as a selection for this item?

File: models/beers.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      flavorProfile: { type: Types.Select, options: 'Malty, Earthy, Aromatic, Sweet, Rich, Delicious', default: 'Delicious', index: true },
+      profileDate: { type: Types.Date, default: Date.now }
+    }
+);
+
+Beers.schema.virtual('content.full').get(function() {
+  return this.content.extended || this.content.brief;
+});
+
+Beers.defaultColumns = 'title, state|20%, author|20%, publishedDate|20%';
```
Comment (caitlin-mohnike) [inline models/beers.js]: Should update these. The beers one doesn't have title, state, etc.

File: models/beers.js (hunk 1 above)
Comment (lizTheDeveloper) [inline models/beers.js]: +1 on caitlin's suggestion, go ahead and fix before merge

File: models/beers.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      brewerName: { type: String, required: true, initial: true},
+      beerImage: {type: Types.CloudinaryImage, required: false, initial: false },
+      isFeaturedBeer: { type: Types.Boolean, required: false}
+    },
+  {heading: 'Beer Profile'},
+    {
+      profileTitle: { type: String },
+      profileAuthor: { type: Types.Name },
+      profileBody: { type: Types.Textarea },
+      flavorProfile: { type: Types.Select, options: 'Malty, Earthy, Aromatic, Sweet, Rich, Delicious', default: 'Delicious', index: true },
```
Comment (lizTheDeveloper) [inline models/beers.js:28]: So, talk to Anders about this, I believe the flavor profile will probably be more complex than a single item. Find out what he wants to track in terms of flavor profile for v1, hit him on Slack

Comment (mattphoto) [inline models/beers.js:28]: i talked to Anders. Agreed that for now, we'll keep it simple. "Let’s just say that the flavor profile is a single item that is made up from the aggregate ratings and move past it".

**Pull request merged**